lein-kibit/src/leiningen/kibit.clj

```diff
@@ -11,21 +11,20 @@
   (let [src-paths     (get-in project [:kibit :source-paths] ["rules"])
         repositories (:repositories project)
         local-repo    (:local-repo project)
-        kibit-project `{:dependencies [[jonase/kibit ~(str/trim-newline
-                                                        (slurp
-                                                          (io/resource
-                                                            "jonase/kibit/VERSION")))]]
+        kibit-version       (clojure.edn/read-string
+                             (slurp (io/resource "version.edn")))
```

Comment:
If any user of kibit has a file called `version.edn` at the root of their classpath then there will be conflicts between this one and that one. It would be good to be able to move it to `jonase/kibit/version.edn` to avoid the risk of conflicts.